src/template.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
+</head>
+<body>
+  <header>
+    <h1>Leaderboard</h1>
+  </header>
+  <main>
+    <section class="leaderboard">
+      <div class="header">
+        <h2>Recent scores</h2>
+        <button type="button">Refresh</button>
+      </div>
+      <ul class="scores">
+        <li>Name: 100</li>
+        <li>Name: 20</li>
+        <li>Name: 50</li>
+        <li>Name: 78</li>
+        <li>Name: 125</li>
+        <li>Name: 77</li>
+        <li>Name: 42</li>
+      </ul>
+    </section>
+    <section class="add-score">
+      <div class="header">
+        <h2>Add your score</h2>
+      </div>
+      <form action="#">
+        <input type="text" name="name" id="name" placeholder="Your name">
+        <input type="number" name="score" id="score" placeholder="Your score">
+        <input type="submit" value="Submit">
+      </form>
+    </section>

```

Comment:
- [ ] Your Html main should contain only the `root element`. It means that your content will be injected by your javascript file by targeting that root element. Let's go ahead and create some functions in your javascript file using arrow functions instead of the `function` keyword to populate your page.

Reply:
Thanks, I've resolved this. the HTML is now rendered dynamically.